ui/src/pages/PasswordReset/PasswordReset.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { Formik, Form as FormikForm } from "formik";
+import * as Yup from "yup";
+import passwordService from "../../services/PasswordService/PasswordService";
+import Form from "react-bootstrap/Form";
+import Container from "react-bootstrap/Container";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import { BiArrowFromBottom } from "react-icons/bi";
+import { usePromise } from "react-use-promise-matcher";
+import FormikInput from "../../parts/FormikInput/FormikInput";
+import FeedbackButton from "../../parts/FeedbackButton/FeedbackButton";
+
+interface PasswordResetParams {
+  password: string;
+  repeatedPassword: string;
+}
+
+const ProfileDetails: React.FC = () => {
+  const validationSchema: Yup.ObjectSchema<PasswordResetParams | undefined> = Yup.object({
```

Comment:
`validationSchema` could be defined outside of the component's body in order to avoid re-creating on each re-render

Reply:
👍